src/wazuh_modules/vulnerability_scanner/tests/unit/exclusionScanner_test.cpp

```diff
@@ -0,0 +1,52 @@
+/*
+ * Wazuh Vulnerability Scanner - Unit Tests
+ * Copyright (C) 2015, Wazuh Inc.
+ * September 21, 2023.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include "exclusionScanner_test.hpp"
+#include "scanOrchestrator/exclusionScanner.hpp"
+#include "scanOrchestrator/scanContext.hpp"
+#include <memory>
+#include <string>
+#include <vector>
+
+/*
+ * @brief Test instantion of the ExclusionScanner class.
+ */
+TEST_F(ExclusionScannerTest, TestInstantionOfTheExclusionScannerClass)
+{
+    // Instantion of the ExclusionScanner class.
+    EXPECT_NO_THROW(std::make_shared<ExclusionScanner>());
+}
+
+/*
+ * @brief Test handleRequest of the ExclusionScanner class.
+ */
+TEST_F(ExclusionScannerTest, TestHandleRequest)
+{
+    std::string data {R"({"type":"dbsync_packages"})"};
+    std::vector<char> message(50, ' ');
+    std::copy(data.begin(), data.end(), message.begin() + 15);
```

Comment:
Not tested but I think this would do the trick
```suggestion
    std::vector<char> message(data.begin(), data.end());
```

Reply:
I'm not really sure about the 50 and 15 meanings...

Reply:
This is repeated several times across all test files.

Reply:
The [ScanContext](https://github.com/wazuh/wazuh/blob/98d1f9ffc20195c8feef247998ccebedf12c6eea/src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/scanContext.hpp#L46) has a `std::vector<char>` as message but for some reason when we run the tests with ASAN it's generating the error `heap-buffer-overflow`. This error occurs when accessing unallocated memory space. That's why I allocated 50 elements in the vector, but it didn't fix the error `heap-buffer-overflow`. It seems to be a false positive generated by ASAN. For now, the best option is to comment the places where the `message` required by the `ScanContext` is used(I added some comments regarding to this). 
Should I remove the tests where I added the variable `message`?